VirtoCommerce.CatalogModule.Web/Scripts/blades/property-list.js

```diff
@@ -14,14 +14,49 @@ angular.module('virtoCommerce.catalogModule')
 		function initialize(entity) {
 			blade.title = entity.name;
 			blade.subtitle = 'catalog.blades.property-list.subtitle';
-			blade.currentEntity = entity;
+            blade.currentEntity = entity;
```

Comment:
The properties filtration based on user selection can be implemented by the simplest way (we have discussed with @AliveMen  abut details)

Reply:
done